node-module-system/add.js

```diff
@@ -0,0 +1,4 @@
+export default function add(one, two) {
+  const sum = one + two;
+  console.log('result:', sum);
```

Comment:
This works, but it would be better to return the sum from this function and log it in your calculate file.